railties/lib/rails/commands/server/server_command.rb

```diff
@@ -86,6 +86,8 @@ def create_tmp_directories
       end
 
       def log_to_stdout
+        raise "cannot log to stdout if daemonized" if options[:daemonize]
```

Comment:
Let's just skip this, once the app daemonizes it's pretty easy to see that nothing will be written to stdout. The option is misspelled too.